inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -374,11 +376,13 @@ static void TransformationUpToCPUSpecificOpSet(std::shared_ptr<ngraph::Function>
     pass_config->disable<ngraph::pass::ConvertReduceMaxToPooling>();
     pass_config->disable<ngraph::pass::ConvertReduceSumToPooling>();
     pass_config->disable<ngraph::pass::SliceToStridedSlice>();
+    pass_config->disable<ngraph::pass::ConvertDetectionOutput8ToDetectionOutput1>();
 
     pass_config->enable<ngraph::pass::NormalizeL2Decomposition>();
     pass_config->enable<ngraph::pass::ConvertInterpolate1ToInterpolate4>();
     pass_config->enable<ngraph::pass::ConvertGather1ToGather7>();
     pass_config->enable<ngraph::pass::ConvertGather8ToGather7>();
+    pass_config->enable<ngraph::pass::ConvertDetectionOutput1ToDetectionOutput8>();
```

Comment:
ConvertDetectionOutput1ToDetectionOutput8 also added in common optimizations. Looks like it is not required here.

Reply:
Reverted